config/environments/production.rb

```diff
@@ -74,15 +74,10 @@
   # Don't log SQL
   config.active_record.logger = nil
 
-  # Logging
-  config.log_level = :info
-
   if ENV["RAILS_LOG_TO_STDOUT"].present?
-    config.rails_semantic_logger.add_file_appender = false
-
     $stdout.sync = true
-
-    config.semantic_logger.add_appender(io: $stdout, level: Rails.application.config.log_level, formatter: :json)
```

Comment:
can we keep this, can see it's not defaulting to `Rails.application.config.log_level` unless I'm wrong?

Reply:
I can see it defaults to `:trace`